starter-code/src/functions-and-arrays.js

```diff
@@ -1,5 +1,17 @@
 // Find the maximum
 
+var num1;
+var num2;
+function maxOfTwoNumbers (num1, num2){
```

Comment:
You don't have to declare `num1` `num2`. 
`maxOfTwoNumbers (num1, num2)` this parameters (num1, num2) are placeholders. You can name it as you want for using inside the function's scope.